glide.lock

```diff
@@ -27,6 +27,8 @@ imports:
   version: 2de1f203e7d5e386a6833233882782932729f27e
 - name: github.com/facebookgo/pidfile
   version: f242e2999868dcd267a2b86e49ce1f9cf9e15b16
+- name: github.com/fortytw2/leaktest
```

Comment:
This is in two places now, figure out what needs to be changed so it's only in one place

Reply:
It is fixed with the latest commit.